demos/device_defender_for_aws/defender_demo.c

```diff
@@ -427,6 +447,36 @@ static BaseType_t collectDeviceMetrics( void )
         }
     }
 
+    /* Collect custom metrics. This demo sends this tasks stack high water mark
+     * as a number type custom metric and the current task ids as a list of
+     * numbers type custom metric. */
+    if( metricsCollectorStatus == MetricsCollectorSuccess )
+    {
+        vTaskGetInfo(
+            /* Query this task. */
```

Comment:
This comment is not relevant to `NULL` parameter value

Reply:
This is relevant, as NULL has it query the current task